content/4_Gold/DSU.mdx

```diff
@@ -41,6 +41,15 @@ export const problems = {
 			['DSU'],
 			'usaco-992'
 		),
+		new Problem(
```

Comment:
I don't think it's worth adding to this module (after all, you can do many of the problems in the DFS module with DSU as well).